rtmp-client/src/main/java/net/butterflytv/rtmp_client/RtmpClient.java

```diff
@@ -131,19 +352,23 @@ private native int nativeOpen(String url, boolean isPublishMode, long rtmpPointe
      *  @throws IOException if connection is not opened or connection to server is lost
      *
      */
-    public int read(byte[] data, int offset, int size) throws IOException {
-        return nativeRead(data, offset, size, rtmpPointer);
+    public int read(byte[] data, int offset, int size) throws IOException, IllegalStateException {
```

Comment:
For the public methods, please add `@throws IllegalStateException if xxxxxx` Javadoc

Reply:
Done.